Milestone5 styling

smcommits opened the pull request:
In this PR, I have applied CSS styling the bookstore CMS. I have followed the provided design guideline on Zeplin. 

### Tools and methods Used
- SASS
- React Styled Components
- React inline styling

### Placeholder Components
There are some components that are just placeholders and are not functional. They are as follows:
- Nav Bar
- Comments
- Edit
- Percentage
- Update Progress

smcommits committed:
Add Style

smcommits committed:
Add Style

smcommits committed:
Update README.md

smcommits committed:
Merge branch 'Milestone5-Styling' of github.com:smcommits/bookstore-cms into Milestone5-Styling

smcommits committed:
Fix Stylelint Errors

smcommits committed:
Fix Eslint Errors

eliasecasta reviewed (changes requested):
# Status:  Changes Required ♻️

Hello, @smcommits,

Great job so far with your app, you are almost done, I will mention you some things that need to be improved in order to approve this project:

- [x] Add all element in design, features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/designAllElementAsPlaceholder.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/)) Please add the [Profile icon](https://prnt.sc/1b4bo5g) missing at your navbar in order to add all the elements as required by Microverse, other than that, your styling looks perfect, great job!
- [x] Please add feedback to your buttons/links when you are hovering over them, right now none of them have it and I can't differentiate between a clickable component and just a placeholder component. What I mean is to display the small hand icon on the mouse pointer when something can be clicked.

Please correct those issues and send another review request, good luck, and well done!

smcommits committed:
Add Profile Icon, Change button hover

Added profile icon in the navbar. The image is svg and the border is applied to the parent container

Added cursor to pointer on interactive links with class="small_text". y

taiwo2 reviewed (approved):
# STATUS: APPROVED 🎉 
@smcommits Congratulations for completing this milestone project and for implementing the suggested changes by the previous Code Reviewer. Your are such a talented developer. your project is approved.
